rocAL/rocAL_pybind/amd/rocal/readers.py

```diff
@@ -22,158 +22,184 @@
 from amd.rocal.pipeline import Pipeline
 import amd.rocal.types as types
 
-def coco(*inputs, file_root, annotations_file='', bytes_per_sample_hint=0, dump_meta_files=False,
-         dump_meta_files_path='', file_list='', initial_fill=1024,  lazy_init=False, ltrb=False, masks=False,
-         meta_files_path='', num_shards=1, pad_last_batch=False, prefetch_queue_depth=1, preserve=False,
-         random_shuffle=False, ratio=False, read_ahead=False, save_img_ids=False, seed=-1, shard_id=0,
-         shuffle_after_epoch=False, size_threshold=0.1, skip_cached_images=False, skip_empty=False,
-         stick_to_shard=False, tensor_init_bytes=1048576, is_box_encoder=False, is_box_iou_matcher=False):
 
+def coco(annotations_file='', ltrb=True, masks=False, ratio=False,
```

Comment:
pad_last_batch and stick_to_shard are we sure this will not be required in the future?

Reply:
Lots of these args are removed please ensure this support will not be required in future.

Reply:
@shobana-mcw Initially the idea was to have params similar to NVIDIA so that, directly changing
fn.readers.coco from DALI pipeline to rocAL pipeline will be easy. The params will remain even though we don't use internally. Is that the intention or we can remove all those not used in rocAL?

Reply:
Added stick_to_shard and pad_last_batch arguments for all readers and reader specific arguments which may be supported in rocal in future